Farmland.py

```diff
@@ -1,205 +0,0 @@
-#This contains the two classes of BarrenLand and FertileLand
-
-import numpy.matlib
-import numpy as np
-
-#class BarrenLand holds information about the barren land,
-class BarrenLand:
-
-    #input is the string with four integers seperated by a space
-    def __init__(self, LowerLeftVertex=-1, UpperRightVertex=-1, stringVertex= '',fieldwidth = 400,fieldheight = 600):

```

Comment:
these variables should be in camelCase.
lowerLeftVertex
upperRightVertext
fieldWidth
fieldHeight